manifests/install.pp

```diff
@@ -1,14 +1,22 @@
 define yarn::install (
   Boolean $production = true,
+  Boolean $frozen_lockfile = $production,
```

Comment:
Initializing a param with another param works, but I have never seen this in other modules…